fud2/scripts/calyx_py-to-calyx.rhai

```diff
@@ -0,0 +1,10 @@
+import "calyx" as c;
```

Comment:
@jku20 Akash has added a fud2 stage that runs `python3` on a file. This spares us from the usual pattern of:
```
python3 myfile.py | 
fud ...
```
Would you please take a look for style etc? Also we know this should have been its own PR; sorry about that!